MapboxCoreNavigation.podspec

```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 123.0"
+  s.dependency "MapboxNavigationNative", "~> 123.1.0"
```

Comment:
This will prevent updates to the 123._X_. I think it'd be better to use:
`s.dependency "MapboxNavigationNative", "~> 123.1"`